app/README-Okta.md

```diff
@@ -0,0 +1,33 @@
+# Using AppAuth with [Okta](https://developer.okta.com/docs/api/resources/oidc.html)
+
+Since Okta is a certified OpenID Connect provider and supports PKCE by default, configuration is quick and simple. There are no additional libraries or tools required.
+
+## Add an OpenID Connect Client
+
+You can create an Okta developer account at [https://developer.okta.com/](https://developer.okta.com/). 
+
+  1. After login, navigate to [https://{{yourOrg}}-admin.oktapreview.com/admin/apps/add-app](https://{{yourOrg}}-admin.oktapreview.com/admin/apps/add-app) and select **Create New App**
```

Comment:
This link won't actually work, so it would probably better just to use an inline code block:

```
After login, navigate to `https://{{yourOrg}}-admin.oktapreview.com/admin/apps/add-app` 
and select **Create New App**
```